### ISSUE TITLE
Split Dataset

### ISSUE THREAD

**Issue by MiguelSteph** [MiguelSteph/heart_disease_prediction#6]
Split the Dataset into training set, validation set and test set
